opendebates/admin.py

```diff
@@ -1,28 +1,52 @@
 import itertools
 
 from django.contrib.admin import ModelAdmin, register
+from django.core.urlresolvers import reverse
 from django.shortcuts import render
+from django.utils.html import format_html
 from django.utils.timezone import now
 from djangohelpers.export_action import admin_list_export
 
 from . import models
 from opendebates_emails.models import send_email
 
 
+def debate_link(debate):
+    return format_html(
+        '<a href="{}">{}</a>'.format(
+            reverse('admin:opendebates_debate_change', args=(debate.pk,)),
+            debate.prefix)
+    )
+
+
 @register(models.Category)
 class CategoryAdmin(ModelAdmin):
-    list_display = [f.name for f in models.Category._meta.fields]
-    actions = [admin_list_export]
+    list_display = ('name', 'prefix')
+    list_filter = ('debate__prefix',)
+    list_select_related = ('debate',)
+
+    actions = (admin_list_export,)
+
+    def prefix(self, obj):
```

Comment:
It kinda bugs me that you have to define `prefix` separately for all of these. … not that I have any suggestions for how to do better.

Does Python let you define parametric mixins so that you could mix in `PrefixedModelAdminMixin('category.debate')` or something like that?